File: app/serena_mcsv/mscv-user/src/main/java/psytobetter/user/mscv_user/dto/user/UserDTO.java
package psytobetter.user.mscv_user.dto.user;

import lombok.Data;

@Data
public class UserDTO {
    private Long id;
    private String userName;
    private String userEmail;
    private String userPassword;
    private Boolean userAceptConditions;
    private String userImageUri;

}

File: app/src/main/java/com/duocuc/serena/data/remote/user/dto/user/UserDTO.kt
package com.duocuc.serena.data.remote.user.dto.user

import com.google.gson.annotations.SerializedName

data class UserDto(
    val id: Long?,

    @SerializedName("userName")
    val userName: String,

    @SerializedName("userEmail")
    val userEmail: String,

    @SerializedName("userPassword")
    val userPassword: String?,

    @SerializedName("userAceptConditions")
    val userAceptConditions: Boolean?,

    @SerializedName("userImageUri")
    val userImageUri: String?
)